src/hotspot/share/gc/g1/g1YoungCollector.cpp

```diff
@@ -983,6 +983,8 @@ void G1YoungCollector::post_evacuate_collection_set(G1EvacInfo* evacuation_info,
 
   allocator()->release_gc_alloc_regions(evacuation_info);
 
+  _evac_failure_regions.prepare_regions();
```

Comment:
Can this task be moved into the `post_evacuate_cleanup_1` phase as a `G1BatchedTask`?